infrap4d/CMakeLists.txt

```diff
@@ -2,13 +2,10 @@ cmake_minimum_required(VERSION 3.12)
 
 project(infrap4d VERSION 0.1.0 LANGUAGES C CXX)
 
-set(STRATUM_TDI_BIN_DIR ${STRATUM_SOURCE_DIR}/stratum/hal/bin/tdi)
-set(STRATUM_BFRT_BIN_DIR ${STRATUM_SOURCE_DIR}/stratum/hal/bin/barefoot)
-
 if(DPDK_TARGET)
-    set(INFRAP4D_MAIN ${STRATUM_TDI_BIN_DIR}/dpdk/main.cc)
+    set(INFRAP4D_MAIN $<TARGET_OBJECTS:stratum_dpdk_o>)
```

Comment:
This is is unacceptable. It undoes the mechanism I designed to support infrap4d. We must supply our own main.cc.

Reply:
Modified to use the main separately.